app/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfig.scala

```diff
@@ -175,4 +175,8 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
   lazy val mfaUpliftUrl: String = config.get[String]("urls.mfaUplift")
 
   lazy val businessAccountUrl: String = config.get[String]("urls.businessAccount")
+
+  lazy val accessibilityStatementServicePath: String =
```

Comment:
`accessibilityStatementServicePath` means something very specific so perhaps leave this variable intact and create another one called something like `grsAccessibilityStatementPath` which prepends "/accessibility-statement" to this one?